packages/react/ds/src/score-select/score-select.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+import React from 'react';
+import {
+  ButtonGroup,
+  ButtonGroupItem,
+} from '../button/button-group/button-group.js';
+import { FormField } from '../forms/form-field.js';
+import { ScoreSelectProps } from './type.js';
+
+export const ScoreSelect: React.FC<ScoreSelectProps> = ({
+  name,
+  size = 'medium',
+  options,
```

Comment:
This is not following the ACs.
We should have a property (e.g. `variant`) that could have 3 possible values:
- `opinion-scale-5`
- `opinion-scale-7`
- `nps`

The component will create the options internally considering the variant selected